doc/introduction/operations.rst

```diff
@@ -473,3 +473,34 @@ Qutrit Observables
 
 :html:`</div>`
 
+.. _intro_ref_ops_pulse:
+
+Pulse-level operators
+---------------------
+
+If you would like to learn more about the implementation of pulse-level control in PennyLane, see the
+:mod:`~.pulse` module documentation.
+
+
+Pulse operator
+^^^^^^^^^^^^^^
+
+:html:`<div class="summary-table">`
+
+.. autosummary::
+    :nosignatures:
+
+    ~pennylane.pulse.ParametrizedEvolution
```

Comment:
I know that `ParametrizedHamiltonian` is not technically an Operation, but I think that it should be in this doc as it's a very important class to pule programming. The significant contrast between `Hamiltonian` and `ParametrizedHamiltonian` makes it somewhat confusing, so having a direct reference to `ParametrizedHamiltonian` here might give the user an easier way to access the documentation for it. The one-line description that renders for `ParametrizedEvolution` does have a reference to it, but I think it might be worthwhile having a separate entry for it. Let me know what you (or any of the other reviewers) think about it

Reply:
I agree 👍